### ISSUE TITLE
Modify JavaScript so Ruby and Svelte are included in Filter on Projects page

### ISSUE THREAD

**Issue by jphamtv** [hackforla/website#6071]
### Overview
Ruby and Svelte languages are being excluded from the 'Languages / Technologies' filter because of a conditional in the `createFilter` function in `assets/js/current-projects.js` file. We need to modify the code so they are included in the 'Languages / Technologies' filter on the Projects and Home pages.

### Action Items
- [ ] Locate the `createFilter` function in the `assets/js/current-projects.js` file
- [ ] Replace the following code:
```
'technologies': [...new Set(sortedProjectData.map(item => (item.project.technologies && item.project.languages?.length > 0) ? [item.project.languages, item.project.technologies].flat() : '').flat() ) ].filter(v=>v!='').sort(),
```
- [ ] With:
```
'technologies': [
    ...new Set(
        sortedProjectData.map(item => {
            let combined = [];
            if (item.project.languages?.length > 0) {
                combined = combined.concat(item.project.languages);
            }
            if (item.project.technologies?.length > 0) {
                combined = combined.concat(item.project.technologies);
            }
            return combined;
        }).flat()
    )
].filter(v => v).sort(),
```
- [ ] Using Docker, test changes on Projects and Home pages to make sure filters are working correctly
- [ ] Confirm Ruby and Svelte display in the 'Languages / Technologies' filter

### Resources/Instructions
[assets/js/current-projects.js ](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js)
[Projects page](https://www.hackforla.org/projects/)
[Home page](https://www.hackforla.org/#)
[See comments where the issue was found](https://github.com/hackforla/website/pull/5978#pullrequestreview-1797662017)
[ER #6274: Modify JavaScript so Ruby and Svelte are included in the Filter on Projects page](https://github.com/hackforla/website/issues/6274)

**Comment by roslynwythe** [comment]
- @jphamtv I think you may need to update this issue, because #6318 is in progress.

**Comment by jphamtv** [comment]
Thank you @roslynwythe, I'll update this to include Tools after #6318 is completed.

**Comment by jphamtv** [comment]
Closing because issue resolved by PR #6467
